setup.sh

```diff
@@ -296,8 +296,14 @@ setup_osie() {
 		mkdir -p "$osie_current"
     	mkdir -p "$tink_workflow"
     	pushd /tmp
-    	curl 'https://tinkerbell-oss.s3.amazonaws.com/osie-uploads/latest.tar.gz' -o osie.tar.gz
-    	tar -zxf osie.tar.gz
+
+        if [ -z $TB_INTERFACE ]; then
```

Comment:
```suggestion
        if [ -z "${TB_OSIE_TAR:-}" ]; then
```

the `${...:-}` says "if this variable is undefined, just make the result empty." Quoting it just in case the path to the tarball has spaces or other spooky characters in it.

Also, it looks like the indentation doesn't quite match. I just opened https://github.com/tinkerbell/tink/pull/143 so that if you your editor configured with editorconfig, it will just indent correctly automatically. Specifically, these lines of code are indented 2 spaces too far, and then the stuff inside the `if` and `else` should be indented with a tab.